dbms/src/Common/getNumberOfLogicalCPUCores.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+/// Get number of CPU cores without hyper-threading.
+unsigned getNumberOfLogicalCPUCores();
```

Comment:
show it in bootstrap https://github.com/pingcap/tiflash/blob/e03dcc1e38e14906e7598c7e8775a2660e9ab58a/dbms/src/Server/Server.cpp#L1403-L1411

Reply:
I will do that in another pr to fix #3821